src/thunderbots/software/param_server/README.md

```diff
@@ -4,11 +4,11 @@ The param server node initializes the dynamic_reconfigure server and calls the `
 
 We currently support `int32_t`, `double`, `bool` and `string`
 ### Registering a parameter to be configured through rqt_reconfigure
-Add the following code in the Params.cfg file found at `param_server/cfg/Params.cfg`
+Add the following code in the ParamArray.cfg file found at `param_server/cfg/ParamArray.cfg`
```

Comment:
Why did the `Params.cfg` filename change here? Accidental refactor?

Reply:
ah, got aggressive with renaming....... whoops, reverted